recipes/tiled/meta.yaml

```
@@ -0,0 +1,316 @@
+{% set name = "tiled" %}
+{% set version = "0.1.0a60" %}
+
+package:
+  name: tiled-suite
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tiled-{{ version }}.tar.gz
+  sha256: 32f3fee3583fe7e899d908d79319c00dd5c20bb5245c44519653c45d20cfc457
+
+build:
+  number: 0
+
+outputs:
+
+  # Categorized requirements
+
+  - name: tiled-array
+    requirements:
+      host:
+        - python
+        - pip
+      run:
+        - python
+        - dask
+        - numpy
+
+  - name: tiled-dataframe
+    requirements:
+      host:
+        - python
+        - pip
+      run:
+        - python
+        - dask
+        - pandas
+        - pyarrow
+
+  - name: tiled-xarray
+    requirements:
+      host:
+        - python
+        - pip
+      run:
+        - python
+        - dask
+        - pandas
+        - pyarrow
+        - xarray
+
+  - name: tiled-compression
+    requirements:
+      host:
+        - python
+        - pip
+      run:
+        - python
+        - blosc
+        - lz4
+
+  - name: tiled-minimal-client
+    script: build-tiled.sh  # [unix]
+    script: bld-tiled.bat  # [win]
+    build:
+      entry_points:
+        - tiled = tiled.commandline.main:main
+    requirements:
+      host:
+        - python
+        - pip
+      run:
+        - python
+        - appdirs
+        - click !=8.1.0
+        - entrypoints
+        - heapdict
+        - httpx >=0.20.0
+        - jsonschema
+        - msgpack-python >=1.0.0
+        - pyyaml
+        - typer
+    test:
+      imports:
+        - tiled
+        - tiled.client
+      commands:
+        - tiled --help
+
+  - name: tiled-minimal-server
+    script: build-tiled.sh  # [unix]
+    script: bld-tiled.bat  # [win]
+    build:
+      entry_points:
+        - tiled = tiled.commandline.main:main
+    requirements:
+      host:
+        - python
+        - pip
+      run:
+        - python
+        - aiofiles
+        - alembic
+        - anyio
+        - cachey
+        - cached-property  # [py<38]
```

I assume this is because you are using cached_property from functools in python >=3.8. Is there any other reason besides this that your package cannot be noarch? Would you consider making your conda package noarch by setting the minimum python version to 3.8 or do you still use python 3.7 at NSLS2?

---

This is the only reason that I am aware of.  We still need python 3.7

---

```suggestion
        # TODO: Can be noarch when Python 3.7 is EOL in June 2023
        - cached-property  # [py<38]
```
That's fine, I suggest a note like above to remind future maintainers.